src/components/Quotes.js

```diff
@@ -1,7 +1,6 @@
 import { useEffect, useState } from 'react';
 // eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] As I pointed out above, and required by MIcroverse, you are not meant to use any Linters disable rule in your codebase. The Linters are there to make sure you follow the best practices. I'll suggest you remove these and try to fix the errors. :smile: :small_airplane: :+1: